src/Restaurant.js

```diff
@@ -1,17 +1,16 @@
 import React from "react";
+import Orders from "./Order";
 
 const Restaurant = () => {
-  const pizzas = 0;
   return (
-    <div>
+    <div id="restautant">
       <h3>Restaurant Orders</h3>
       <ul>
-        <li>
-          Pizzas: {pizzas} <button className="btn btn-primary">Add</button>
-        </li>
+        <Orders orderType={"Pazzia"} />
```

Comment:
Can you add css to these buttons so that they can look a bit tidier, please?
![image](https://user-images.githubusercontent.com/61394003/207153885-b66b5263-98a6-4776-abe6-1abe9a908715.png)

Reply:
I will do , Thanks for reviewing my code